src/OnlineRetailPortal.Web/Validations/UploadImageRequestValidator.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.AspNetCore.Http;
+using FluentValidation;
+using OnlineRetailPortal.Core;
+using OnlineRetailPortal.Web.Models;
+using System.IO;
+
+namespace OnlineRetailPortal.Web.Validations
+{
+    public class UploadImageRequestValidator : AbstractValidator<HttpRequest>
+    {
+        public UploadImageRequestValidator()
+        {
+            RuleFor(req => req)
+           .Cascade(CascadeMode.StopOnFirstFailure)
+           .NotNull()
+           .WithErrorCode(StatusCodes.Status400BadRequest.ToString())
+           .WithMessage(Error.MissingImage());
+
+
+
+            RuleFor(req => req.Form)
+            .Cascade(CascadeMode.StopOnFirstFailure)
```

Comment:
why CascadeMode if the rule has only one validation ?

Reply:
Yes, Removed